fed/mosaic-cell/src/main/java/org/eclipse/mosaic/fed/cell/config/util/ConfigurationReader.java

```diff
@@ -75,7 +76,8 @@ public static CNetwork importNetworkConfig(String networkConfPath) throws Intern
 
         // server capacity isn't limited by the network configuration, but handled within the cell module configuration
         networkConfig.servers.forEach((server) -> {
-            if (server.downlink.capacity != 0 || server.uplink.capacity != 0) {
+            if (ObjectUtils.defaultIfNull(server.downlink.capacity, 0L) != 0 ||
```

Comment:
Couldn't we just compare with `null` now?

Reply:
I guess so.